ui/app/components/sidebar/nav/cluster.js

```diff
@@ -22,14 +22,15 @@ export default class SidebarNavClusterComponent extends Component {
     return this.namespace.inRootNamespace && !this.cluster?.hasChrootNamespace;
   }
 
-  get showSync() {
-    // Only show sync if cluster is not managed
-    return this.flags.managedNamespaceRoot === null;
-  }
+  get badgeText() {
+    const isManaged = this.flags.isManaged;
+    const onLicense = this.version.hasSecretsSync;
```

Comment:
nit: Since these two variables are only used once, I think it reads a little easier to use them directly in the conditional which more clearly relays which service it comes from. Just a thought 💭

Reply:
Totally understand, but I also like the way this reads. My brain sees these const names and knows exactly what conditions I'm going to be iterating through. Whereas the service getters (e.g. `hasSecretsSync` doesn't really tell me that it's checking the license vs a flag).